speedcord/shard.py

```diff
@@ -79,6 +92,9 @@ async def read_loop(self):
                 self.logger.warning("Unknown message type: " + str(type(message)))
 
     async def send(self, data: dict):
+        """
+        Attempts to send a message via the gateway. Checks for the gateway ratelimit before doing so. 
```

Comment:
Why do you say attempts here? Also there is no reason to include ratelimits here in my opinion.
Also where is :param: here?